carbon-design-system/carbon-for-ibm-dotcom
# fix(masthead-v2): fix masthead-contact & add DDO for testing

### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/9360

### Description

- This PR fixes the issue where the "Contact" CTA is not initializing the contact module's Let's Talk module when the `has-contact` attribute is set to `true`.

### Changelog

**Changed**

- Update the `Masthead` component CodeSandbox to include a test `DDO` and `cm-app.min.js` script so that we can verify that the behavior is working as expected.


https://user-images.githubusercontent.com/1815714/189696785-e00a7997-7c33-4322-ad6a-008eefacda45.mov

### Testing Instructions
- Visit the `Masthead` codesandbox app - https://webcomponents-codesandbox.s3-web.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/9359/masthead/index.html
- Click on the "Contact" CTA and ensure that the "Let's talk" widget opens.
